cfme/common/provider.py

```diff
@@ -56,6 +56,7 @@ class BaseProvider(WidgetasticTaggable, Updateable, Navigatable):
     STATS_TO_MATCH = []
     db_types = ["Providers"]
     ems_events = []
+    settings_key = None
```

Comment:
This I'm iffy about, since I tried to be safe about the settings_key calls anyway. :man_shrugging: